donkeycar/pipeline/types.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 from donkeycar.parts.tub_v2 import Tub
-from donkeycar.utils import load_image_arr, normalize_image, train_test_split
+from donkeycar.utils import load_image_arr, load_pil_image, normalize_image, train_test_split
```

Comment:
I think this might be unnecessary (unless there is a specific optimization you are going after).  We could just add a `raw` option to `load_image` to load the raw PIL image if that's what you were going for. 

Otherwise, `Image.from_array(...)` can be used to construct a `PIL` `Image` type from a `numpy` array. That might be what you want to do in the pipeline itself. Also add a note on channel ordering for those who might be unfamiliar with the incoming shapes.

For the reverse, you could do a `np.as_array(pil_image)`.

Reply:
I was trying to avoid an unnecessary conversion from `PIL` -> `numpy` -> `PIL` -> `tensor`. I wasn't sure modifying `load_image_arr` to also return a raw image made sense, since the function name specifies loading an array.

Could I change `load_image_arr` to be named `load_image` and then have an option whether to return a `numpy` array or a `PIL` image?

Reply:
Yeah.. lets make that change. Lets have the following:

`load_pil_image` returns a `Image`. 
`load_image` converts a PIL image -> numpy.ndarray using `np.as_array(load_pil_image(...))`. 

That way you don't have to do back and forth.

Reply:
Sounds good. Just pushed again with these changes.